script/process_images.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Process and convert svg images to png using palette colors."""
+
+# Standard library imports
+from __future__ import absolute_import, division, print_function
+import os
+import sys
+import tempfile
+
+# Third party imports
+from qtpy.QtCore import QSize
+from qtpy.QtGui import QIcon
+from qtpy.QtWidgets import QApplication
+
+# Local imports
+from qdarkstyle.qss import DarkPalette
+
+# Constants
+HERE = os.path.abspath(os.path.dirname(__file__))
+ROOT_PATH = os.path.dirname(HERE)
+SVG_PATH = os.path.join(ROOT_PATH, 'svg')
+RC_PATH = os.path.join(ROOT_PATH, 'qdarkstyle', 'rc', )
+BASE_COLOR = '#ff0000'
+
+
+def get_file_color_map(fname, palette=DarkPalette):
+    """Return map of files to color from given palette."""
+    color_normal = palette.COLOR_FOREGROUND_DARK
+    color_focus = palette.COLOR_SELECTION_LIGHT
+    color_disabled = palette.COLOR_BACKGROUND_NORMAL
+    files_map = {
+        'checkbox_checked.svg': {
+            'checkbox_checked.svg': color_normal,
+            'checkbox_checked_focus.svg': color_focus,
+            'checkbox_checked_disabled.svg': color_disabled,
+        },
+        'checkbox_indeterminate.svg': {
+            'checkbox_indeterminate.svg': color_normal,
+            'checkbox_indeterminate_focus.svg': color_focus,
+            'checkbox_indeterminate_disabled.svg': color_disabled,
+        },
+        'checkbox_unchecked.svg': {
+            'checkbox_unchecked.svg': color_normal,
+            'checkbox_unchecked_focus.svg': color_focus,
+            'checkbox_unchecked_disabled.svg': color_disabled,
+        },
+        'radio_checked.svg': {
+            'radio_checked.svg': color_normal,
+            'radio_checked_focus.svg': color_focus,
+            'radio_checked_disabled.svg': color_disabled,
+        },
+        'radio_unchecked.svg': {
+            'radio_unchecked.svg': color_normal,
+            'radio_unchecked_focus.svg': color_focus,
+            'radio_unchecked_disabled.svg': color_disabled,
+        }
+    }
+    for f, file_colors in files_map.items():
+        if f == fname:
+            break
+
+    assert file_colors
+
+    return file_colors
+
+
+def create_colored_svg(svg_path, temp_svg_path, base_color, new_color):
+    """Replace base svg with new fill color."""
+    with open(svg_path, 'r') as fh:
+        data = fh.read()
+
+    new_data = data.replace(base_color, new_color)
+
+    with open(temp_svg_path, 'w') as fh:
+        fh.write(new_data)
+
+
+def convert_svg_to_png(path_svg, path_png, height, width):
+    """Convert svg files to png files using Qt."""
+    size = QSize(height, width)
+    icon = QIcon(path_svg)
+    pixmap = icon.pixmap(size)
+    img = pixmap.toImage()
+    img.save(path_png)
+
+
+def main():
+    """"""
+    app = QApplication([])
```

Comment:
What is this QAPP for? It is needed?

Reply:
Hmm habit I guess. Not sure if the qicons work without it

Reply:
Because of `QPixmap`

```
QPixmap: Must construct a QGuiApplication before a QPixmap
Abort trap: 6
```